### ISSUE TITLE
MenuToggle - add aria-invalid and aria-autocomplete

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#11806]
**Describe the enhancement or change**
Per https://github.com/patternfly/patternfly/issues/6562#issuecomment-2839684312:

- MenuToggle should be updated to apply `aria-invalid="true"` when the status is danger for non-typeahead/plain toggles (basic toggles and split button toggles). If there are any typeahead examples showing status, those should be updated to apply the aria-invalid to the input element.
- For typeahead variants, examples and templates should be updated so that `aria-autocomplete="list"` is applied to the text input being used as the typeahead input.

**Is this request originating from a Red Hat product team? If so, which ones and is there any sort of deadline for this enhancement?**

**Any other information?**
